### ISSUE TITLE
Updating `about/home`

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/hdbscan-feedstock#58]
The current `about/home` is...

https://github.com/conda-forge/hdbscan-feedstock/blob/7587bf978a3eb68b60f11f57148e7728af562045/recipe/meta.yaml#L43

However it looks like this has moved to https://github.com/scikit-learn-contrib/hdbscan

The PyPI package also points to the same location

Should `about/home` be updated?

**Comment by lmcinnes** [comment]
Yes, sorry. I am travelling right now, but I can try to get to this when I get back.

**Comment by jakirkham** [comment]
Thanks Leland! 🙏

No worries. Was looking at something else in the recipe and spotted this. So just wanted to surface it for awareness

**Comment by jakirkham** [comment]
Done in PR: https://github.com/conda-forge/hdbscan-feedstock/pull/59
